gen/calc.py

```diff
@@ -1089,6 +1091,8 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'check_search_path': CHECK_SEARCH_PATH,
         'mesos_master_work_dir': '/var/lib/dcos/mesos/master',
         'mesos_agent_work_dir': '/var/lib/mesos/slave',
+        'mesos_agent_log_file': '/var/log/mesos/mesos-agent.log',
+        'mesos_master_log_file': '/var/lib/dcos/mesos/log/mesos-master.log',
```

Comment:
Because these two entries are in the `default` map, these will potentially be exposed to a user's configuration.  Meaning the user would be able to specify `mesos_master_log_file` and move it to a different location.

Not sure if this is intentional, but we probably want to move these into the `must` mapping below instead.

Reply:
Fixed